fp.js

```diff
@@ -18,7 +18,7 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(function(person) {
+people.filter(function(person) {
```

Comment:
Echoing my `.map` comment, although in this case you just need to return the condition everything in the resulting array should meet.

```js
let peopleYoungerThan35 = people.filter(function(person){
  return person.age < 35
})
```